File: dmr-api/Models/LunchTime.cs
using System;
using System.Collections.Generic;
using System.ComponentModel.DataAnnotations.Schema;
using System.Linq;
using System.Threading.Tasks;
using dmr_api.Data.Interface;

namespace DMR_API.Models
{
    public class LunchTime: IDateTracking
    {
        public int ID { get; set; }
        public DateTime StartTime { get; set; }
        public DateTime EndTime { get; set; }

        public int CreatedBy { get; set; }
        public int DeletedBy { get; set; }
        public int UpdatedBy { get; set; }

        public int IsDelete { get; set; }
        public DateTime CreatedTime { get; set; }
        public DateTime? DeletedTime { get; set; }
        public DateTime? UpdatedTime { get; set; }
    }
}

File: dmr-app/src/app/_core/_model/lunch.time.ts
export interface LunchTime {
    iD: number;
    startTime: Date;
    endTime: Date;
    createdBy: number;
    deletedBy: number;
    updatedBy: number;
    isDelete: number;
    createdTime: string;
    deletedTime: string | null;
    updatedTime: string | null;
}
